test/functional/helpers.lua

```diff
@@ -6,8 +6,10 @@ local AsyncSession = require('nvim.async_session')
 local Session = require('nvim.session')
 
 local nvim_prog = os.getenv('NVIM_PROG') or 'build/bin/nvim'
+-- FIXME: 'autoindent' messes up the insert() function
+-- 'laststatus=1' makes the screen related tests less cluttered
```

Comment:
It is understood that these `--cmd` settings are meant to work around and/or enforce the expectations of the tests, so I removed these comments.